### PR TITLE
Azure Security Center - Adding support for Python, TypeSrcipt and Node.js

### PR DESCRIPTION
**Pull request by chlahav**
### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/security/resource-manager/readme.nodejs.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-security
+  output-folder: $(node-sdks-folder)/lib/services/SecurityManagement
```
Comment (daschult) [inline specification/security/resource-manager/readme.nodejs.md]: Please make `SecurityManagement` camel-cased (`securityManagement`).

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by jianghaolu** [comment]
LGTM - will merge once the comment from daschult is resolved.

File: specification/security/resource-manager/readme.nodejs.md (hunk 1 above)
Comment (chlahav) [reply specification/security/resource-manager/readme.nodejs.md]: @daschult changed to `securityManagement`

**Review by ghost — APPROVED** [review]

**Comment by chlahav** [comment]
@jianghaolu the comment from daschultis was resolved and he approved.
if you don't have additional comments, can you please merge?
thanks!

**Pull request merged**